Add basic automated exception capturing for Ruby (#9143)
getsentry/sentry-docs: platform-includes/getting-started-config/ruby.mdx

  end
end
```

> Note that this will only set up Sentry, but **not** capture any exceptions automatically.
> Setting up automatic exception capturing is usually done by adding the Sentry extension
> corresponding to your framework. If you cannot find a Sentry extension which meets your needs,
> you can add the following code.

Reviewer: ```suggestion
The code above will only set up Sentry, **not** capture any exceptions automatically.
Setting up automatic exception capturing is usually done by adding the Sentry extension
corresponding to your framework. If you cannot find a Sentry extension which meets your needs,
you can add the following code:
```
